apps/nacl_demos/nacl_demos.cpp

```diff
@@ -207,6 +210,30 @@ class HalideDemosInstance : public Instance {
             halide_set_num_threads(thread_pool_size);
         }
 
+        if (0) {
+            // Print the layout of buffer_t and exit
+            std::ostringstream oss;
+            oss << "sizeof(halide_buffer_t) = "    << sizeof(halide_buffer_t) << "<br>";
+            oss << "sizeof(halide_dimension_t) = " << sizeof(halide_dimension_t) << "<br>";
+            oss << "sizeof(halide_type_t) = "      << sizeof(halide_type_t) << "<br>";
+            halide_buffer_t b;
```

Comment:
It would be super-awesome if halide_buffer_t had a default operator<< overload for people to use for debugging.